build/workspace.bzl

```diff
@@ -110,7 +110,7 @@ def debian_image_dependencies():
             digest = _digest(_DEBIAN_BASE_DIGEST, arch),
             registry = "k8s.gcr.io",
             repository = "debian-base",
-            tag = "0.4.1",  # ignored, but kept here for documentation
+            tag = "1.0.0",  # ignored, but kept here for documentation
```

Comment:
i'm pretty sure the shas above need to be updated, but that is pointless until we push the images anyhow

Reply:
(lines 91-96, not sure how we are populating these exactly)

Reply:
the debian base and debian-iptables digests maps above need to be updated with the values in 
https://github.com/kubernetes/kubernetes/pull/82966#issuecomment-548981763
https://github.com/kubernetes/kubernetes/pull/82966#issuecomment-548999120
these are in the same order except the overall manifest is at the bottom.
amd64, arm, arm64, ppc64le, s390x. the digest is the hash at the end of the line after "digest" 🙃 

also, this is `v2.0.0` now, right? (even if it is ignored)